pentaho-karaf-assembly/src/main/filtered-resources/etc/custom.properties

```diff
@@ -274,7 +274,7 @@ org.osgi.framework.system.packages.extra= \
  org.pentaho.di.job.entry, \
  org.pentaho.di.www, \
  org.springframework.security;version=2.0.5.RELEASE, \
- org.springframework.security.context, \
+ org.springframework.security.context;version=2.0.5.RELEASE, \
```

Comment:
@pentaho-nbaker kept the same spring-security version that is being applied on the rest of the org.springframework.security packages in this file: 2.0.5.RELEASE;

There is another custom.properties file, in etc-server, that does indeed reference version 2.0.8.RELEASE; 

If this etc/custom-properties needs updating the version from 2.0.5 to 2.0.8, then an overall update to all spring.security packages exposed here should be made; would you like me to go ahead and do that ?